.kokoro/release.sh

```diff
@@ -1,6 +1,59 @@
 #!/bin/bash
 
-# Make sure `JAVA_HOME` is set.
-echo "JAVA_HOME=$JAVA_HOME"
+# Get secrets from keystore and set and environment variables.
+setup_environment_secrets() {
+  export SONATYPE_USERNAME=functions-framework-release-bot
+  export SONATYPE_PASSWORD=$(cat ${KOKORO_KEYSTORE_DIR}/75669_functions-framework-java-release-bot-sonatype-password)
+  export GPG_PASSPHRASE=$(cat ${KOKORO_KEYSTORE_DIR}/75669_functions-framework-java-release-bot-gpg-passphrase)
+
+  export GNUPGHOME=/tmp/gpg
+  mkdir $GNUPGHOME
+  mv ${KOKORO_KEYSTORE_DIR}/75669_functions-framework-java-release-bot-gpg-pubring $GNUPGHOME/pubring.kbx
```

Comment:
where does this get used? Could you add a comment if this is some magic-path that needs to be correct for something to work.

Reply:
Done.